### ISSUE TITLE
loss function？

### ISSUE THREAD

**Issue by wjxxyz** [h-y1heng/StableMoFusion#13]
You described the definition of the loss function in 4.4 Footskate Reduction in your paper, but I can't find it in the code, can you please point out the path to the specific file?

**Comment by h-y1heng** [comment]
The loss function for training vGRFs, function 5 in the paper, is implemented in [scripts/train_UnderPressure_model.py line128](https://github.com/h-y1heng/StableMoFusion/blob/53ec611216e37cf7156ab47c6f3e9f3301a77b4a/scripts/train_UnderPressure_model.py#L128).  The loss function for removing the foot slip in the generated action by iterative optimization, function 5 in the paper, is implemented in [utils/footskate_clean.py line 122](https://github.com/h-y1heng/StableMoFusion/blob/53ec611216e37cf7156ab47c6f3e9f3301a77b4a/utils/footskate_clean.py#L122)
